test/lit.cfg

```diff
@@ -878,7 +878,7 @@ if run_vendor == 'apple':
     config.target_sdk_name = xcrun_sdk_name
     config.target_ld = "%s ld -L%r" % (xcrun_prefix, make_path(test_resource_dir, config.target_sdk_name))
     config.target_swift_frontend = (
-        "%s -frontend %s -sdk %r %s %s " %
+        "%s -frontend %s -sdk %r %s %s -ignore-module-source-info" %
```

Comment:
Please add this to *all* compiler substitutions. Right now you've skipped the mock SDK ones *and* the Linux ones, which makes me think `target_options` would be a better place to make the change.